raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -190,7 +190,34 @@ library NettingChannelLibrary {
         self.closed = block.number;
     }
 
-    /// @notice close(bytes, bytes) to close a channel between to parties
+
+    /// @notice close a channel between two parties without a raiden transfer
+    ///         Cases where this may need to happen is:
+    ///         - Alice deposits but Bob doesn't, then Alice must be able to
+    ///           close the channel and reclaim her deposit.
+    ///         - Neither Alice or Bob have made any deposits
+    ///         - Generally any case where you want to simply close the channel
+    ///           without adding an extra transfer to it
+    /// @param caller_address The address of the participant trying to close
```

Comment:
Isn't this covered by:

> Alice deposits but Bob doesn't, then Alice must be able to
> close the channel and reclaim her deposit

I can also add a comment for single funded channels there.